### PR TITLE
Add a default value for the /myentries command

### LINKED ISSUE #6: Default value for myentries
I'm too lazy to write `/myentries 5` each time.

### PR DESCRIPTION
**Pull request by JesseWilliamson**
Fixes #6

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -149,10 +149,14 @@ def user_entries(ack, respond, body, command, logger):
     try:
         user_id = body['user_id']
         name = full_name(user_id)
-        num_entries = int(command['text'])
+        argument = command['text']
+        if argument == "":
+            num_entries = 5
+        else:
+            num_entries = int(argument)
```
Comment (CMurtagh-LGTM) [inline app.py]:
```suggestion
        num_entries = int(command['text']) if command['text'] != "" else 5
```
I think this looks nicer as a ternary

File: app.py [hunk 2]
```diff
@@ -149,10 +149,10 @@ def user_entries(ack, respond, body, command, logger):
     try:
         user_id = body['user_id']
         name = full_name(user_id)
-        num_entries = int(command['text'])
+        num_entries = int(command['text']) if command['text'] != "" else 5
```
Comment (TrentHouliston) [inline app.py:152]: Well I think it looks better as a default get!
```suggestion
        num_entries = int(command.get('text', 5))
```

Comment (TrentHouliston) [reply app.py:152]: that is unless it's actually an empty string in the dictionary, then don't do this

Comment (JesseWilliamson) [reply app.py:152]: If the user just does the command with no other text there will still be an empty string in the dictionary so I don't think this works

**Review by CMurtagh-LGTM — APPROVED** [review]
LGTM

**Pull request merged**